Libraries/pbxspec/Sources/PBX/Specification.cpp

```diff
@@ -38,6 +38,18 @@ Specification()
 }
 
 bool Specification::
+operator==(Specification const &rhs) const
+{
+    return _identifier == rhs._identifier && _domain == rhs._domain;
```

Comment:
My understanding is that if we do this right, we shouldn't have any duplicate identifier/domain couples for specifications, so effectively, comparing the ID compares the spec itself.

That being said, I'm fine with not using `operator==` for this, but I've got really no idea what the method should be named, so I'll take suggestions.